pixeltable/exec/data_row_batch.py

```diff
@@ -16,14 +16,11 @@ class DataRowBatch:
 
     tbl: Optional[catalog.TableVersionHandle]
     row_builder: exprs.RowBuilder
-    img_slot_idxs: list[int]
-    media_slot_idxs: list[int]  # non-image media slots
-    array_slot_idxs: list[int]
     rows: list[exprs.DataRow]
 
     def __init__(
         self,
-        tbl: Optional[catalog.TableVersionHandle],
+        tbl: catalog.TableVersionHandle,
```

Comment:
Pls put `assert tbl is not None` in the init body just to convince me that this is correct